lib/cfg-grammar.y

```diff
@@ -892,7 +893,11 @@ block_arg
           LL_BLOCK
           {
             cfg_lexer_pop_context(lexer);
-            cfg_args_set(last_block_args, $1, $3); free($1); free($3);
+            if ($3 && cfg_lexer_is_backtick_substitution_postponed(lexer) && strchr($3, '`'))
+              cfg_args_set(last_block_args, $3, $3);
```

Comment:
I don't understand the first cfg_args_set call. It sets the arg name $3 with the same value as the key? That's the body of the argument, so in this case:
```
block foo(arg1(alma) arg2(`arg1`)) {

}
```
The first arg doesn't have a backtick, so we'd store "arg1"="alma", the second arg does have a backtick, so we would store "`arg1`"="`arg1`" and arg2="`arg1`"

What's the point in storing the backticked version?

Can you show an example? Thanks

Reply:
backtick in a key can only happens if the backtick substitution is postponed - so the `cfg_lexers_subst_invoke`  can check if a value is stored in the block_args as name - if it is stored, then backtick subst is postponed